test/Spring-test.js

```diff
@@ -58,19 +60,14 @@ describe('Spring', () => {
     TestUtils.renderIntoDocument(<App />);
   });
 
-  it('should allow a defaultValue', done => {
+  it('should allow a defaultValue', () => {
     let count = [];
     const App = React.createClass({
       render() {
         return (
-          <Spring defaultValue={{val: 1}} endValue={{val: 10}}>
+          <Spring defaultValue={{val: 0}} endValue={{val: 10}}>
             {({val}) => {
```

Comment:
To keep track of all the values given to the child-as-function of the `Spring`. Or is that not your question?